build.gradle

```diff
@@ -1,4 +1,5 @@
 // For those who want the bleeding edge
+//file:noinspection JCenterRepository
```

Comment:
What does this do?

Reply:
Jcenter will close in 2022 and intellij warns about it, forgot it disabled the inspection directly in the file

Reply:
in my experience, removing jcenter with our repositories does nothing besides cleaning up the useless warning, so you can safely remove it